joinmarket/message_channel.py

```diff
@@ -255,7 +255,7 @@ def privmsg(self, nick, cmd, message, mc=None):
             self.active_channels[nick].privmsg(nick, cmd, message)
             return
         else:
-            log.debug("Failed to send message to: " + str(nick) + \
+            log.warn("Failed to send message to: " + str(nick) + \
```

Comment:
Don't mind either way, assumed this doesnt happen often since I have not seen that message yet. I'd favor info.